functions/dcWaiter.R

```diff
@@ -31,28 +33,49 @@ dcWaiter <- function(stage = c("show", "update", "hide"),
         ),
         color = "#424874"
       )
-    } else if (isPass) {
+    # } else if (!isLogin) {
+    #   # when user is not login
+    #   waiter_update(html = tagList(
+    #     img(src = "img/synapse_logo.png", height = "120px"),
+    #     h3("Looks like you're not logged in!"), 
+    #     span("Please ", 
+    #       a("login", href = "https://www.synapse.org/#!LoginPlace:0", target = "_blank"),
+    #       " to Synapse, then refresh this page."
+    #     )
+    #   ))
+    } else if (!isCertified) {
+      # when user is not certified synapse user
       waiter_update(html = tagList(
         img(src = "img/synapse_logo.png", height = "120px"),
-        h3(sprintf("Welcome, %s!", usrName))
+        h3("Looks like you're not a certified synapse user!"),
+        span("Please follow the ", 
+          a("instruction", 
+            href = "https://help.synapse.org/docs/User-Account-Tiers.2007072795.html#UserAccountTiers-CertifiedUsers", 
+            target = "_blank"
+          ),
+          " to become a certified user, then refresh this page."
+        )
+      ))
+    } else if (!isPermission) {
+      # when user is not certified synapse user
+      waiter_update(html = tagList(
+        img(src = "img/synapse_logo.png", height = "120px"),
+        h3("Looks like you dont have access to fileview"),
+        span("placeholder ...")
```

Comment:
@milen-sage I need your help on this one.

My testing account do not have access to the filevie. I will get this error and app will crash:
```
Warning: Error in py_call_impl: AccessCredentialsError: Your access to 'syn20446927'' could not be resolved. Please check your credentials and try again.

Detailed traceback:
File "/home/rchai/miniconda3/envs/data_curator_dev_env/lib/python3.7/site-packages/schematic/store/synapse.py", line 83, in **init**
raise AccessCredentialsError(self.storageFileview)
```

I would like to prevent the app crash and show the error message in the initial loading screen. It could possibly happen if users not have access to fileview?

The approach I added here is to notify that they don't have access to the fileview, but I don't know what description/instructions to help them solve this issue. 

preview
--- 
<img width="769" alt="Screen Shot 2021-11-17 at 10 57 24 AM" src="https://user-images.githubusercontent.com/73901500/142242577-40f1ad4b-05d9-4d90-980e-c418beb7f28e.png">